_data/pages/es/events.yml

```diff
@@ -7,6 +7,16 @@ intro:
 
 upcomingEvent:
 
+  - title: Open Distro for Elasticsearch - Community Meeting
+    content: "Join us for our our biweekly online community meeting"
+    link: "https://www.meetup.com/Open-Distro-for-Elasticsearch-Meetup-Group/events/thmcwrybcnbhb/"
+    location: "Online"
+    date: 10/5 2020
```

Comment:
Let's start using ISO-8601 date format.

Reply:
Going forward or should I adjust past events as well?